src/OpenTK/Platform/Utilities.cs

```diff
@@ -235,11 +235,26 @@ public static IWindowInfo CreateX11WindowInfo(IntPtr display, int screen, IntPtr
         /// Creates an IWindowInfo instance for the windows platform.
         /// </summary>
         /// <param name="windowHandle">The handle of the window.</param>
+        /// <param name="flags">Graphics context flags.</param>
         /// <returns>A new IWindowInfo instance.</returns>
-        public static IWindowInfo CreateWindowsWindowInfo(IntPtr windowHandle)
+        public static IWindowInfo CreateWindowsWindowInfo(IntPtr windowHandle, GraphicsContextFlags flags = GraphicsContextFlags.Default)
```

Comment:
Here we would want to pass as little information as needed, so I would rather we pass a single bool here or an enum. It could be very confusing for users to be able to specify multiple options that don't have any effect on the results of the function.